packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -1684,6 +1704,28 @@ export class StripeHelper {
     return plans;
   }
 
+  async allConfiguredPlans(): Promise<ConfiguredPlan[]> {
+    // for a transitional period we will include configs from both Firestore
+    // docs and Stripe metadata, making it possible for Payments to toggle
+    // the Firestore configs feature flag without any changes or re-deploy
+    // necessary on the auth-server
```

Comment:
@biancadanforth This is the main reason I didn't put everything behind the feature flag.  Since auth-server only reads the config docs, it seems safe.

Reply:
Hmm, but if we're making a request on the payments server to the /plans endpoint, the auth server will try to construct this `configuration` prop's value regardless of any flags on the payments server right?

I mentioned this in another comment, but my concern is that this affects the /plans endpoint which we hit on basically every page load on payments.